addon/services/scroll-activity.js

```diff
@@ -57,7 +58,8 @@ export default Service.extend(Evented, {
   },
 
   _pollScroll() {
-    if (window.requestAnimationFrame) {
+	if (!self.window.requestAnimationFrame) { return; }
```

Comment:
What does this do, exactly? This seems like it would break the flow where we want to support browsers that don't have rAF by using `setTimeout`.